ksql-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsStateStore.java

```diff
@@ -38,29 +39,30 @@
   private final String stateStoreName;
   private final KafkaStreams kafkaStreams;
   private final LogicalSchema schema;
-  private final Duration timeout;
+  private final KsqlConfig ksqlConfig;
   private final Supplier<Long> clock;
 
   KsStateStore(
       final String stateStoreName,
       final KafkaStreams kafkaStreams,
-      final LogicalSchema schema
+      final LogicalSchema schema,
+      final KsqlConfig ksqlConfig
   ) {
-    this(stateStoreName, kafkaStreams, schema, DEFAULT_TIMEOUT, System::currentTimeMillis);
```

Comment:
`DEFAULT_TIMEOUT` isn't used anymore and should be removed

Reply:
Removed